### ISSUE TITLE
add note about Rollbar.silenced to DelayedJob integration docs

### ISSUE THREAD

**Issue by lexelby** [rollbar/rollbar-gem#510]
I have a DelayedJob job that I don't want rollbar to report exceptions for.  However, I _do_ want exceptions from all other jobs to get logged to rollbar.  The docs make it unclear how to do this:

https://github.com/rollbar/rollbar-gem#delayedjob

The code doesn't have any obvious points to indicate that a given job shouldn't have exceptions reported (I was thinking maybe I could set an instance method on a custom job class?).  Turns out the answer is [`Rollbar.silenced`](https://github.com/rollbar/rollbar-gem#silencing-exceptions-at-runtime) which is hidden in an entirely unrelated section of the docs.

A link from the DelayedJob integration docs to the `Rollbar.silenced` docs would save others the time I just spent digging around for a solution. :)

**Comment by jondeandres** [comment]
@lexelby we don't have a specific and official way to not report an error in a DelayedJob job. However you found a way you can use for it. You could just wrap your code in a block `Rollbar.silenced do`.

We have a section in the README dedicated to `Rollbar.silenced`, https://github.com/rollbar/rollbar-gem#silencing-exceptions-at-runtime. I'm not sure we should add that link in all sections in the documentation.

**Comment by lexelby** [comment]
I did find that section of the docs (linked above), but only after I had decided on the terribly hacky method of using a dynamic level filter to silence the exceptions.  I was expecting to have to dig around in the Scope object to figure out if the exception was from the specific delayed job I wanted to silence.

I understand not wanting to link it everywhere, but as a (relatively new) rollbar-gem user, it was pretty unclear to me how to silence any specific delayed job's exceptions, and some kind of hint in the docs would really have helped.
